client/src/pages/treedata/TreeCare.js

```diff
@@ -7,7 +7,7 @@ import { treeHealth } from '@/util/treeHealth';
 export default function TreeCare({
   currentTreeId, common, health,
 }) {
-  const { data: treeHistory } = useTreeHistoryQuery({ currentTreeId });
+  const { data: treeHistory } = useTreeHistoryQuery({ id: currentTreeId });
```

Comment:
One thing we could do is change the custom hooks to take just the `id` as a param, instead of a whole param object.  That way, the param name wouldn't be repeated everywhere, and the caller wouldn't have to care what it is.

Reply:
I just meant passing a bare number instead of an object, like:

> const { data: treeHistory } = useTreeHistoryQuery(currentTreeId);

That way, you don't need to care about how that is sent to the server.  For multiple params, it would need to be an array, like `useTreeLikesQuery([idTree, user.email])`.  Of course, that gets more complicated with multiple params, as then the order matters, so not sure how helpful it would be.